src/rust/devcontainer-feature.json

```diff
@@ -57,7 +57,7 @@
                 "vadimcn.vscode-lldb",
                 "rust-lang.rust-analyzer",
                 "tamasfe.even-better-toml",
-                "serayuzgur.crates"
+                "fill-labs.dependi"
```

Comment:
Instead of replacing, can we simply remove the crates extension? 

We should wait for user feedback and do a security check before adding a dependency on `fill-labs.dependi`

Reply:
ok sure